snuba/utils/streams/topics.py

```diff
@@ -29,6 +29,7 @@ class Topic(Enum):
     DEAD_LETTER_METRICS = "snuba-dead-letter-metrics"
     DEAD_LETTER_SESSIONS = "snuba-dead-letter-sessions"
     DEAD_LETTER_GENERIC_METRICS = "snuba-dead-letter-generic-metrics"
+    DEAD_LETTER_REPLAYS = "snuba-dead-letter-replays"
```

Comment:
You will also need to add the topic to the config validation rules.
https://github.com/getsentry/snuba/blob/master/snuba/settings/validation.py#L21-L46

Reply:
@fpacifici Forgot to push after my last comment.  Should be good to go.